connection-service/new-vm-join-domain/new-vm-join-domain.json

```diff
@@ -133,19 +126,13 @@
                         "version": "latest"
                     },
                     "osDisk": {
-                        "name": "osdisk",
-                        "vhd": {
-                            "uri": "[concat('http://',parameters('storageAccountName'),'.blob.core.windows.net/vhds/',parameters('dnsLabelPrefix'), 'disk0.vhd')]"
-                        },
+                        "name": "[concat(parameters('dnsLabelPrefix'), '-os')]",
                         "caching": "ReadWrite",
                         "createOption": "FromImage"
                     },
                     "dataDisks": [
                         {
-                            "name": "myvm-data-disk1",
```

Comment:
I'm sure this is from me eon's ago but I was never fully certain why we need the extra disks.

Reply:
Maybe the default OS disk size is too small?

Reply:
Yeah, I didn't really see the point either, but didn't bother changing it. Default size is 128 GB.

Reply:
According to https://docs.microsoft.com/en-us/azure/virtual-machines/windows/about-disks-and-vhds , maximum size for OS disk is 2 TB so maybe that's why

Reply:
2 TB is pretty big :) Okay with a 128 GB default, and the increased pricing of managed storage, I think we can nuke the second drives.